subprojects/core-api/src/main/java/org/gradle/api/artifacts/repositories/MavenArtifactRepository.java

```diff
@@ -119,7 +119,7 @@
          * Indicates that this repository will contain Maven POM files.
          * If the POM file contains a marker telling that Gradle metadata exists
          * for this component, Gradle will <i>also</i> look for the Gradle metadata
-         * file.
+         * file. Redirect will not happen if ignoreGradleMetadataRedirection is present
```

Comment:
I suggest applying the same suggestions as for the Ivy javadoc here and on the added method.